sshuttle/ssh.py

```diff
@@ -116,7 +116,7 @@ def connect(ssh_cmd, rhostport, python, stderr, options):
         if python:
             pycmd = "'%s' -c '%s'" % (python, pyscript)
         else:
-            pycmd = ("P=python3.5; $P -V 2>/dev/null || P=python; "
+            pycmd = ("P=python3; $P -V 2>/dev/null || P=python; "
```

Comment:
Not sure about this line. Will it break if python3.4 is installed on the server? The server code might actually be 3.4 compatable. Then again I would like to pretend nobody is still using python3.4 :-).

Reply:
Yes, server code can run with all versions of Python from 2.4 to 3.6. Just to be sure I will test again tomorrow and let you know before merging.